src/javascript/app/pages/trade/barriers.js

```diff
@@ -198,6 +198,17 @@ const Barriers = (() => {
         });
     };
 
+    const isAbsoluteZero = num =>  {
+        if (parseFloat(num) !== 0) {
+            return false;
+        } else if (Object.is(parseFloat(num), -0)) {
+            return false;
+        } else if (num.toString().charAt(0) === '+') {
```

Comment:
why not do `/+|-/.test(num.toString().charAt(0))` and remove the second condition?